### ISSUE TITLE
About pretrained HourGlass model

### ISSUE THREAD

**Issue by qibao77** [Maclory/Deep-Iterative-Collaboration#7]
How to pretrain the HourGlass model? Can you provide more details? Thank you!

**Comment by Steve-Tod** [comment]
Hi, we pretrain the HourGlass on CelebA dataset.
For training details, we follow [this git repo](https://github.com/raymon-tian/hourglass-facekeypoints-detection) and represent the 68 landmarks as 68-channel heatmap.
Since we are using a Feedback HourGlass, we take HR Image and feedback feature of previous step as input during each recurrent step. We aggregate L2 losses between output heatmap and ground truth heatmap of all the steps and optimize the network.

**Comment by qibao77** [comment]
Thank you for your reply! What's more, is the ground truth heatmap obtained by OpenFace?

**Comment by Steve-Tod** [comment]
Yes. We use OpenFace to get ground truth landmarks and generate the heatmaps accordingly.

**Comment by JingzheLyp** [comment]
Hi, I've trained the  Feedback HourGlass model on CelebA according to your method, but the loss is very large about 1e+24, the init means is 'kaiming'. When I change the init means from 'kaiming'  to 'normal', the loss decrease from 1e-1 to 1e-4, about 1.1W steps ,then it become to nan. @Steve-Tod ,  can you provide more details? such like net config, init and learning strategy, Thank you!
